health/drivers/utils.py

```diff
@@ -15,11 +15,14 @@
 
 import datetime
 import json
+import logging
 
 import requests
 
 
 TS_FMT = "%Y-%m-%dT%H:%M:%S"
+LOGGING_FORMAT = '[%(asctime)s] %(levelname)s in %(module)s: %(message)s'
```

Comment:
This is wrong place to specify LOGGING_FORMAT 

I would prefer to put it in better place (e.g. closer to entry point -> job.py file)